src/widgetastic/widget.py

```diff
@@ -1173,6 +1173,10 @@ def fill(self, value):
             else:
                 self.logger.info('Filling column %r', key)
 
+            # if the row widgets aren't visible the row needs to be clicked to edit
+            if not self[key].widget.is_displayed:
+                self.click()
+                changed = True
```

Comment:
@mfalesni Conditional on a parameter to fill()? Would need kwargs passed through view fill in that case yes?

Understood about potentially misleading 'changed' value, but I think clicking a row in the table (given that action actually displays column widgets) constitutes a 'change' to the form.  Especially when this return is used by `Table.fill` to call `Table.row_save`.  Even if none of the widgets in that row were changed, the row was selected for changing and needs to be saved.

NOTE
This is setting `changed` in a loop, and overwriting itself. Which means if I have 3 columns, and the last one filled doesn't change, the change in the first or second column filled is not recorded! This seems like significant bug to me, and I'm pretty sure I've seen it in action in MIQ AnalysisProfiles testing of this implementation.